dashboard/src/shared/url.ts

```diff
@@ -160,8 +153,8 @@ export const api = {
   },
 
   operators: {
-    operatorIcon: (cluster: string, namespace: string, name: string) =>
-      `api/v1/clusters/${cluster}/namespaces/${namespace}/operator/${name}/logo`,
+    operatorIcon: (_cluster: string, namespace: string, name: string) =>
```

Comment:
Not sure about the current multicluster support for operators, but given the upcoming changes, doesn't really matter I guess.

Reply:
Yes, in the multicluster scenario we are assuming now that operators logos live in the main (Kubeapps) cluster.